lib/framework/procbot.ts

```diff
@@ -45,34 +44,31 @@ interface ServiceMap<T> {
  * * Retrieve configuration file from either the local FS or a ServiceEmitter
  */
 export class ProcBot {
+	/**
+	 * Retrieve the binary path for the Node dependencies.
+	 * @returns Promise containing a string denoting the absolute path.
+	 */
+	public getNodeBinPath = _.memoize((): Promise<string> => {
```

Comment:
There's no `resolver` parameter (I'm only passing the function used), so it doesn't apply.